raster-foundry/raster-foundry
# Scaffold out project layers frontend routes / components

## Overview
Create components and routes for upcoming project layers tasks

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo
![image](https://user-images.githubusercontent.com/4392704/51850738-59775900-22f0-11e9-9580-19868d9e5f8b.png)

### Notes
Used component based routes since they make things a bit simpler. 


## Testing Instructions

* Visit each route and verify that it loads the component correctly
* Verify that routes which should redirect do so.

Closes #4526
